en/setup/fast-rtps-installation.md

```diff
@@ -1,13 +1,21 @@
-# Fast RTPS installation ![](../../assets/fastrtps/eprosima_logo.png)
+# Fast RTPS installation 
 
-[_eprosima Fast RTPS_](http://eprosima-fast-rtps.readthedocs.io/en/latest/) is a C++ implementation of the RTPS (Real Time Publish Subscribe) protocol, which provides publisher-subscriber communications over unreliable transports such as UDP, as defined and maintained by the Object Management Group (OMG) consortium. RTPS is also the wire interoperability protocol defined for the Data Distribution Service (DDS) standard, again by the OMG.
+<img src="../../assets/fastrtps/eprosima_logo.png" style="float:left;"/> [eProsima Fast RTPS](http://eprosima-fast-rtps.readthedocs.io/en/latest/) is a C++ implementation of the RTPS (Real Time Publish Subscribe) protocol, which provides publisher-subscriber communications over unreliable transports such as UDP, as defined and maintained by the Object Management Group (OMG) consortium. RTPS is also the wire interoperability protocol defined for the Data Distribution Service (DDS) standard, again by the OMG.
 
-**NOTE**: The information here was extracted from original [_eprosima Fast RTPS_ documentation](http://eprosima-fast-rtps.readthedocs.io/en/latest/)
+> **Tip** Fast RTPS is used by PX4 to enable publish/subscribe of uOrb topics to be extended to offboard components, including robotics and simulator tools. For more information see: [Micro RTPS](../middleware/micrortps.md).
```

Comment:
@hamishwillee 
1. A more accurate description is found in the beginning of micrortps.md
2. I changed DDS for RTPS
3. We need a link to our readthedocs